docs/installing_agones.md

```diff
@@ -0,0 +1,210 @@
+[vms]: https://cloud.google.com/compute/docs/instances/
+
+### Creating the firewall
+
+We need a firewall to allow UDP traffic to nodes tagged as `game-server` via ports 7000-8000.
+
+```bash
+gcloud compute firewall-rules create game-server-firewall \
+  --allow udp:7000-8000 \
+  --target-tags game-server \
+  --description "Firewall to allow game server udp traffic"
+```
+
+Continue to [Enabling creation of RBAC resources](#enabling-creation-of-rbac-resources)
+
+# Setting up a Minikube cluster
+
+This will setup a [Minikube](https://github.com/kubernetes/minikube) cluster, running on an `agones` profile.
+
+## Installing Minikube
+
+First, [install Minikube][minikube], which may also require you to install
+a virtualisation solution, such as [VirtualBox][vb] as well.
+
+[minikube]: https://github.com/kubernetes/minikube#installation
+[vb]: https://www.virtualbox.org
+
+## Creating an `agones` profile
+
+Let's use a minikube profile for `agones`.
+
+```bash
+minikube profile agones
+```
+
+## Starting Minikube
+
+The following command starts a local minikube cluster via virtualbox.
+
+```bash
+minikube start --kubernetes-version v1.9.0 --vm-driver virtualbox \
+  --extra-config=apiserver.Admission.PluginNames=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota \
+  --extra-config=apiserver.Authorization.Mode=RBAC
+```
+
+# Enabling creation of RBAC resources
+
+To install Agones, a service account needs permission to create some special RBAC resource types.
+
+```bash
+# Kubernetes Engine
+kubectl create clusterrolebinding cluster-admin-binding \
+  --clusterrole cluster-admin --user `gcloud config get-value account`
+# Minikube
+kubectl create clusterrolebinding cluster-admin-binding \
+  --clusterrole=cluster-admin --serviceaccount=kube-system:default
+```
+
+# Installing Agones
+
+Finally, we install Agones to the cluster.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/googlecloudplatform/agones/release-0.1/install.yaml
+```
+
+## Confirming Agones started successfully
+
+To confirm Agones is up and running, run the following command:
+
+```bash
+kubectl describe --namespace agones-system pods
+```
+
+It should describe the single pod created in the `agones-system` namespace, with no error messages or status. The `Conditions` section should look like this:
+
+```
+Conditions:
+  Type           Status
+  Initialized    True 
+  Ready          True 
+  PodScheduled   True
+```
+
+That's it! This creates the [Custom Resource Definitions][crds] that power Agones and allows us to define resources of type `GameServer`.
+
+[crds]: https://kubernetes.io/docs/concepts/api-extension/custom-resources/
+
+# What's next
+
+* Go through the [Create a Game Server Quickstart][quickstart]
+
+[quickstart]: docs/create_gameserver.md
```

Comment:
Broken link - should be `create_gameserver.md` (since it is already in the docs folder)

Reply:
damn it. Done.